src/modules/flow/iio/iio.json

```diff
@@ -218,6 +218,12 @@
              "name": "offset"
          },
          {
+             "data_type": "direction-vector",
```

Comment:
In iio.json, data_type cannot be sol_str_table here, please check https://github.com/solettaproject/soletta/blob/master/src/lib/common/include/sol-types.h